### PR TITLE
ec2utils convert to Py3

### PR DESCRIPTION
**Pull request by rjschwei**

### CODE REVIEW

File: ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2015 SUSE LLC, Robert Schweikert
+# Copyright 2017 SUSE LLC, Robert Schweikert
 #
 # This file is part of ec2uploadimg.
 #
```
Comment (smarlowucf) [inline ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py:4]: For consistency use absolute imports. L26-27

File: ec2utils/ec2utilsbase/lib/ec2utils/ec2utils.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 # along with ec2utilsbase.  If not, see <http://www.gnu.org/licenses/>.
 
 import boto3
-import ConfigParser
+import configparser
 import os
 
 from .ec2UtilsExceptions import *
```
Comment (smarlowucf) [inline ec2utils/ec2utilsbase/lib/ec2utils/ec2utils.py]: Ditto absolute imports.

**Review by smarlowucf — CHANGES_REQUESTED** [review]
Looks good, just a couple imports to fix.

File: ec2utils/ec2uploadimg/ec2uploadimg [hunk 3]
```diff
@@ -383,13 +396,20 @@ try:
...
+                        file=sys.stderr
+                    )
                     sys.exit(1)
             else:
-                print >> sys.stderr, 'Could not reliable determine the ',
-                print >> sys.stderr, 'bootkernel to use '
-                print >> sys.stderr, 'must specify bootkernel, ',
-                print >> sys.stderr, 'arch (x86_64|i386) or hvm'
+                print(
+                    'Could not reliable determine the ',
```
Comment (bear454) [inline ec2utils/ec2uploadimg/ec2uploadimg]: TODO: replace "reliable" with "reliably" (not really apropos to this PR - just noticed it.)

**Review by bear454 — APPROVED** [review]

**Review by smarlowucf — APPROVED** [review]

**Pull request merged**